prdoc/pr_9821.prdoc

```diff
@@ -0,0 +1,17 @@
+title: 'wasmtime: support for perfmap added'
+doc:
+- audience: Node Dev
+  description: |-
+    This PR add  support for `perfmap` in wasmtime executor.
+
+    For more technical details refer to this [doc](https://docs.wasmtime.dev/examples-profiling-perf.html#profiling-with-perfmap).
+
+    Instruction on how to configure profiling on substrate nodes (tested with cumulus benchmarks) is [here](https://hackmd.io/o_Ghc86OT4KzCE4x04MeOg?view#Getting-the-right-flamegraph).
```

Comment:
Nice one! I'd love it if we could find a place to preserve this link (maybe a readme somewhere?) so we can refer to it anytime we need this profiling 🙏